tools/packaging/packaging.go

```diff
@@ -40,18 +40,20 @@ func CreatePackages(
 	insecureGrpc,
 	autoupdate bool,
 	updateChannel string,
+	control bool,
+	controlHostname string,
 	identifier string,
 	omitSecret bool,
 	systemd bool,
 	certPins,
 	rootPEM string,
 ) (*PackagePaths, error) {
-	macPkgDestinationPath, err := CreateMacPackage(packageVersion, osqueryVersion, hostname, secret, macPackageSigningKey, insecure, insecureGrpc, autoupdate, updateChannel, identifier, omitSecret, certPins, rootPEM)
+	macPkgDestinationPath, err := CreateMacPackage(packageVersion, osqueryVersion, hostname, secret, macPackageSigningKey, insecure, insecureGrpc, autoupdate, updateChannel, control, controlHostname, identifier, omitSecret, certPins, rootPEM)
```

Comment:
a PackageParams struct is starting to look pretty good :P

Reply:
yeah, one day...